app/assets/stylesheets/components/_header-notice.scss

```diff
@@ -18,12 +18,21 @@
   padding: govuk-spacing(4);
 }
 
+.app-c-header-notice__text {
+  margin-bottom: govuk-spacing(2);
```

Comment:
Don't need that new fancy fangled responsive mixin here?
:D

Reply:
@huwd 10px doesn't get smaller on lower breakpoints and I think it would be too small if it did, so I think it's ok as is.